sys/net/gnrc/conn/gnrc_conn.c

```diff
@@ -52,11 +52,33 @@ static inline void _srcport(uint16_t *port, gnrc_pktsnip_t *hdr)
 int gnrc_conn_recvfrom(conn_t *conn, void *data, size_t max_len, void *addr, size_t *addr_len,
                        uint16_t *port)
 {
+    return gnrc_conn_recvfrom_timeout(conn, data, max_len, addr, addr_len, port, CONN_NO_TIMEOUT);
```

Comment:
Sorry, I messed up the markdown above.

I see the need for recvfrom() with timeout, I just don't get why the function does something three times, some not-and'ing, and then peeks into the message queue, compared to ```xtimer_set_msg_timeout(); msg_recv();```.